plugins/json_importer.cc

```diff
@@ -250,12 +245,9 @@ struct JSONImporter: public Procedure {
                 return handleError(exc.what(), actualLineNum, string(line, lineLength));
             }
 
-            if (useWhere) {
-                JsonScope jsonScope(server, expr);
-                auto whereBound = config.where->bind(jsonScope);
-                if (!whereBound(row, storage, GET_ALL).isTrue()) {
-                    return true;
-                }
+            JsonRowScope row(expr);
```

Comment:
Put this inside an if (useWhere) { ... } so that we don't execute if there is no where clause.